### PR TITLE
Add profile page CSS and endpoint to delete account

### PR DESCRIPTION
**Pull request by matt989253**

### CODE REVIEW

File: src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            authenticationFacade.unregister(UnregisterCommand.builder()
+                    .username(req.getParameter("username"))
+                    .password(req.getParameter("password"))
+                    .build());
+
+            String redirect = (String) req.getSession().getAttribute("redirectUrl");
+            redirect = (redirect != null)
+                    ? redirect
+                    : getServletContext().getContextPath() + "/questions";
+            resp.sendRedirect(redirect);
```
Comment (alexandrepiveteau) [inline src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java]: Don't we want to always redirect them to the home / list of questions ?

**Review by alexandrepiveteau — CHANGES_REQUESTED** [review]

File: src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java (hunk 1 above)
Comment (matt989253) [reply src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java]: Good point

Comment (alexandrepiveteau) [inline src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java]:
```suggestion
            String redirect = getServletContext().getContextPath() + "/questions";
            resp.sendRedirect(redirect);
```

Comment (alexandrepiveteau) [reply src/main/java/ch/heigvd/amt/stack/ui/web/endpoints/profile/UnregisterCommandEndpoint.java]: Added as a suggestion.

**Review by alexandrepiveteau — APPROVED** [review]

**Pull request merged**